config/discourse_defaults.conf

```diff
@@ -174,6 +174,9 @@ s3_access_key_id =
 s3_secret_access_key =
 s3_use_iam_profile = false
 s3_cdn_url =
+s3_backup_bucket =
```

Comment:
we don't need to add these though... enable_s3_uploads is implied if it is specified via globals

Reply:
Yes, but it looks like if a site doesn't specify those settings in globals then `s3_config_check` is going blow up with method undefined errors. Also tests are failing with the same error if I remove those.